### PR TITLE
Fewer DB queries while loading progress

### PR DESCRIPTION
**Pull request by balderdash**

### CODE REVIEW

File: dashboard/app/helpers/users_helper.rb [hunk 1]
```diff
@@ -140,13 +140,14 @@ def summarize_user_progress_for_all_scripts(user)
   # Since this is currently just used for multi-page LevelGroup levels, we only check that a valid
   # (though not necessarily correct) answer has been given for each level embedded on a given page.
   def get_pages_completed(user, sl)
-    level = user.last_attempt_for_any(sl.levels).try(:level) || sl.oldest_active_level
-
-    if level.is_a? LevelGroup
+    # Since we only swap LevelGroups with other LevelGroups, just check levels[0]
```
Comment (ashercodeorg) [inline dashboard/app/helpers/users_helper.rb:143]: Can this be restructured to return nil early, removing the if statement? E.g.,

```
def get_pages_completed(user, sl)
  return nil unless sl.levels[0].is_a? LevelGroup

  ...
end
```

Comment (balderdash) [reply dashboard/app/helpers/users_helper.rb:143]: Yep, that's nicer. Done.

**Comment by ashercodeorg** [comment]
LGTM.

**Comment by balderdash** [comment]
Circle failure is just that publicKeyCryptography eyes failure. Merging.

**Pull request merged**